### ISSUE TITLE
Text inputs fields cannot be identified as the text border is not visible in high contrast mode

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4633]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
- Test URL: https://clarity.design/documentation/datalist
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Turn on high contrast mode (press left Shift, left alt and print screen keys together).
2. Open the test URL.
3. Observe the data list input fields.

**Expected result:**
All the elements on the page should be displayed as per the high contrast settings theme.

**Actual result:**
Data list text input fields border is not visible in high contrast mode, which makes it difficult or not possible to identify it as text input field.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
